### DISCUSSION TITLE
I cannot Verify my phone Number?

### DISCUSSION THREAD

**Discussion by shadowDarsan** [vercel/community#2359] [Help] [answered]
### Summary

I am trying to make an account and It is not letting me verify my phone number. 
Every time I get this error , 
Error: The verification phone number has been temporarily blocked. Please wait 12 hours and try again.

**Comment by amyegan** [comment, marked as answer]
Hi @shadowDarsan. If you have not already done so, please email [registration@vercel.com](mailto:registration@vercel.com) and include your username. They'll be able to help you get this sorted 🙇
